deploy/main/50-ds-ca.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: connection-agent
+  namespace: example-com
+spec:
+  selector:
+    matchLabels:
+      app: connection-agent
+  template:
+    metadata:
+      labels:
+        app: connection-agent
+      annotations:
+        prometheus.io/sample: "true"
+        prometheus.io/port: "9294"
+    spec:
+      serviceAccountName: connection-agent
+      nodeSelector:
+        role.kos.example.com/workload: "true"
+      hostNetwork: true
+      dnsPolicy: "ClusterFirstWithHostNet"
+      containers:
+      - name: connection-agent
+        image: mattedocker/kos-connection-agent:latest
```

Comment:
After the initial trials, we probably want to pin down a specific commit as opposed to `latest` to know exactly what we're testing.
Also, we don't need to support multiple docker repositories, so I hardcoded mine (`mattedocker`). But I'm not sure whether this meets the secuirty requirements we have, and I read your email on the old automation that you had, where you set up a local docker registry. We will be doing the same, right?

Reply:
The same observations apply to occurrences of the docker image `mattedocker/kos-connection-agent:latest` in other manifests.